packages/nocodb/src/lib/constants/index.ts

```diff
@@ -1,4 +1,5 @@
 export const NC_LICENSE_KEY = 'nc-license-key';
 export const NC_APP_SETTINGS = 'nc-app-settings';
-export const NC_ATTACHMENT_FIELD_SIZE =
-  +process.env['NC_ATTACHMENT_FIELD_SIZE'] || 20 * 1024 * 1024; // 20 MB
+export const NC_ATTACHMENT_FIELD_SIZE = +(
+  process.env['NC_ATTACHMENT_FIELD_SIZE'] || 20 * 1024 * 1024
+); // 20 MB
```

Comment:
Won't work as expected if `NC_ATTACHMENT_FIELD_SIZE` is a wrong value. For example, if the value is `abc` then the final output will be `NaN`. If `NaN` won't cause any issues then it's fine.

Reply:
I think it should be `+process.env['NC_ATTACHMENT_FIELD_SIZE'] || 20 * 1024 * 1024` instead.